engine-src/Game/LambdaHack/Client/UI/HandleHumanGlobalM.hs

```diff
@@ -1716,11 +1716,11 @@ challengesMenuHuman cmdSemInCxtOfKM = do
       width = if isSquareFont propFont then 42 else 84
       duplicateEOL '\n' = "\n\n"
       duplicateEOL c = T.singleton c
-      blurb = splitAttrString width $ textToAS $ T.concatMap duplicateEOL
-              $ mdesc gameMode
-                <> if T.null (mnote gameMode)
-                   then "\n"  -- whitespace compensates for the lack of note
-                   else "\n[Note: " <> mnote gameMode <> "]"
+      blurb = splitAttrString width $ textToAS 
+              $ mmotivation gameMode
```

Comment:
`duplicateEOL` was needed, because sometimes the text has many paragraphs and in this context it looks better if the paragraph is marked by empty line, not just a line break. So it could be better to put here

    T.concatMap duplicateEOL (mmotivation gameMode <> "\n")

or, equivalently

    T.concatMap duplicateEOL (mmotivation gameMode) 
    <> "\n\n"

Reply:
okay how is this 
`      blurb = splitAttrString width $ textToAS
        $ T.concatMap duplicateEOL (mmotivation gameMode <> "\n")
        <> mrules gameMode
`

Reply:
It's almost perfect, but I'd indent it differently (there are many good alternatives, but this is not one of them). Four good options:

    blurb = splitAttrString width $ textToAS $
      T.concatMap duplicateEOL (mmotivation gameMode <> "\n")
      <> mrules gameMode

    blurb = splitAttrString width $ textToAS
            $ T.concatMap duplicateEOL (mmotivation gameMode <> "\n")
              <> mrules gameMode

    blurb = splitAttrString width $ textToAS
              (T.concatMap duplicateEOL (mmotivation gameMode <> "\n")
               <> mrules gameMode)

    blurb = splitAttrString width $ textToAS $
              T.concatMap duplicateEOL (mmotivation gameMode <> "\n")
              <> mrules gameMode